src/legacy/status_im/ui/screens/advanced_settings/views.cljs

```diff
@@ -145,7 +154,9 @@
                   network-name                     [:network-name]
                   transactions-management-enabled? [:wallet-legacy/transactions-management-enabled?]
                   current-log-level                [:log-level/current-log-level]
-                  current-fleet                    [:fleets/current-fleet]]
+                  current-fleet                    [:fleets/current-fleet]
+                  is-sepolia-enabled?              [:profile/is-sepolia-enabled?]
```

Comment:
This line needs to be removed as it's not used and we have enabled Sepolia as the default test network in https://github.com/status-im/status-mobile/pull/18917.